src/pallene/ir.lua

```diff
@@ -51,7 +51,8 @@ function ir.Function(loc, name, typ)
         captured_vars = {},   -- list of ir.VarDecl
         f_id_of_upvalue = {}, -- { u_id => integer }
         f_id_of_local = {},   -- { v_id => integer }
-        body = false,         -- ir.Cmd
+        blocks = {},          -- { ir.BasicBlock }
+        ret_vars = {},        -- { v_id }, list of return variables
```

Comment:
I think the argument vars are specified implicitly, but here we're specifying the return vars explicitly. Is this intentional? It is better to be implicit or explicit?

Reply:
It's intentional. The argument vars are supposed to always be the first on the vars list. We could establish further that the next vars are the return vars, but I find it easier to just have a list with the return vars ids. It's more straight forward to access them and we don't have to fear breaking things if changing the order in which variables are inserted on the function.